pages/static/src/js/groups.js

```diff
@@ -141,12 +180,23 @@ module.exports = function() {
 
         // TODO: show recurring meeting here instead
         description = groups[i].properties.description;
+        group_type = groups[i].properties.group_type;
+        
+        if (group_type == 'state-organizing-committee') {
+          group_type = 'State Organizing Committee';
```

Comment:
Can we access the model we made for this? Otherwise seems not very DRY to write out display name again here.

Reply:
Methods like `get_foo_display` aren't available with current implementation, otherwise agree.

Reply:
Hmm, I'm not sure how to access the Model from a js file. I guess something to think about/research for future.